pkg/plugins/policies/meshaccesslog/api/v1alpha1/validator.go

```diff
@@ -30,8 +30,10 @@ func validateTop(targetRef common_api.TargetRef) validators.ValidationError {
 			common_api.MeshGateway,
 			common_api.MeshService,
 			common_api.MeshServiceSubset,
+			common_api.Dataplane,
 		},
 		GatewayListenerTagsAllowed: true,
+		IsInboundPolicy:            true,
```

Comment:
we have to take this info from the ResourceDescriptor, otherwise we might forget to set it for new policies

Reply:
changed